app/controllers/shipments_controller.rb

```diff
@@ -1,5 +1,13 @@
+require 'aftership'
+
 class ShipmentsController < ApplicationController
   def index
     @shipments = Shipment.all
   end
+
+  def show
+    @shipment = Shipment.find(params[:id])
+    AfterShip.api_key = ENV['AFTERSHIP_API_KEY']
```

Comment:
how you store this key ?

Reply:
I use [figaro](https://github.com/laserlemon/figaro) for storing this key